source/operations/cert-manager.rst

```diff
@@ -0,0 +1,577 @@
+ 
+  While cert-manager supports `earlier K8s versions <https://cert-manager.io/docs/installation/supported-releases/>`__, the MinIO Operator requires a currently supported version.
+- `kustomize <https://kustomize.io/>`__ installed
+- ``kubectl`` access to your ``k8s`` cluster
+
+Getting Started
+---------------
+
+Setup cert-manager
+~~~~~~~~~~~~~~~~~~
+
+Install cert-manager. 
+`Release 1.12.X LTS <https://cert-manager.io/docs/releases/release-notes/release-notes-1.12/>`__ is preferred, but you may install latest.
+
+The following command installs version 1.12.13 using ``kubectl``.
+
+.. code-block:: shell
+   :class: copyable
+   
+   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.13/cert-manager.yaml
+
+For more details on installing cert-manager, see their `installation instructions <https://cert-manager.io/docs/installation/>`__.
+
+Create a self-signed root Issuer for the cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``Cluster Issuer`` is the top level Issuer from which all other certificates in the cluster derive. 
+
+1. Request cert-manager to generate this by creating a ``ClusterIssuer`` resource.
+
+   Create a file called ``selfsigned-root-clusterissuer.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+   
+      # selfsigned-root-clusterissuer.yaml
+      apiVersion: cert-manager.io/v1
+      kind: ClusterIssuer
+      metadata:
+        name: selfsigned-root
+      spec:
+        selfSigned: {}
+
+2. Apply the resource to the cluster:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f selfsigned-root-clusterissuer.yaml
+
+.. _minio-operator-services-with-cert-manager:
+
+MinIO Operator services with cert-manager
+-----------------------------------------
+
+MinIO Operator manages the TLS certificate issuing for the services hosted in the ``minio-operator`` namespace. 
+That is the :ref:`Secure Token Service (sts) <minio-sts-operator>`.
+
+This section describes how to generate the ``sts`` TLS certificate with cert-manager.
+
+- These certificates **must** be issued *before* installing Operator.
+- The cluster's self-signed root ``ClusterIssuer`` certificate must already exist, as described above.
+
+Secure Token Service (STS)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO STS is a service included with MinIO Operator that provides Native IAM Authentication for Kubernetes. 
+This service allows you to control access to your MinIO tenant from your kubernetes applications without having to explicitly create credentials for each application. 
+For more information, see the :ref:`Secure Token Service documentation <minio-sts-operator>`.
+
+Think of the STS Service as a webserver presented with a TLS certificate for ``https`` traffic.
+This guide covers how to **disable** the automatic generation of the certificate in MinIO Operator and issue the certificate using cert-manager instead.
```

Comment:
So I'm not super familiar with this, but is it _absolutely required_ to talk about the STS certificate here? I feel like it will confuse users, when all they really need to know is "If you set up cert-manager for MinIO Operator, you can use those certificates for authenticating to the Operator" or something.

Reply:
I cut out most of this.